pubspec.yaml

```diff
@@ -27,8 +27,8 @@ environment:
   flutter: ">=3.10.0"
 
 dependencies:
-  dio: ^5.4.3
+  dio: ^5.7.0
   flutter:
     sdk: flutter
-  flutter_map: ^7.0.0
-  meta: ^1.11.0
+  flutter_map: ^7.0.2
+  meta: ^1.15.0
```

Comment:
`meta: ^1.15.0` could cause dependency issues with older flutter versions.
Similar bug reports:
- https://github.com/fleaflet/flutter_map/issues/1896
- https://github.com/fleaflet/flutter_map/issues/1910

Reply:
Totally forgot about that, sorry 😬. Fixed with #10 and v3.0.2 (v3.0.1 retracted).